### DISCUSSION TITLE
Add `meta` as a query param of REST API `/v6/deployments`, like CLI `list`

### DISCUSSION THREAD

**Discussion by jjangga0214** [vercel/vercel#8416] [Ideas] [answered]
CLI docs: https://vercel.com/docs/cli#commands/list/extended-usage

`meta` can be passed to CLI's `list` command. 

<img width="1021" alt="스크린샷 2022-08-17 오후 4 00 38" src="https://user-images.githubusercontent.com/28584151/185055298-a3c03d6f-b5db-4e44-a016-47795231b678.png">

However, REST API `/v6/deployments` does not have such a query parameter.

REST API docs: https://vercel.com/docs/rest-api#endpoints/deployments/list-deployments

Thus hereby I request the feature.

Thanks :)

**Comment by amyegan** [comment, marked as answer]
Thanks for sharing your idea! I'm adding it to list of requested features.

**Reply by jjangga0214** [reply]
@amyegan  Hi!
This hasn't been implemented yet.
May I ask if this idea is on a to-do list or just ignored?
Thanks.

**Reply by emmanuelvacher** [reply]
Hello @jjangga0214,

It seems not to be documented but you _can_ filter by `meta` (I've used this feature just today).

Below is an example `curl` command that filters deployment by `meta.githubCommitSha` (automatically set by the GitHub integration) and `meta.foo` (user-defined):

```
curl -XGET -H "Authorization: Bearer $VERCEL_TOKEN" \
  https://api.vercel.com/v6/deployments?teamId=<TEAM_ID>&app=<PROJECT_ID>&meta-githubCommitSha=<COMMIT_SHA>&meta-foo=barr
```

**Reply by jjangga0214** [reply]
@emmanuelvacher Wow, great! How did you find the way? Guesswork?

@amyegan This should be definitely documented!
